### PR TITLE
modify some configs

### PR DESCRIPTION
**Pull request by szy21**
## Purpose 

- Adds implicit diffusion config for `default_perf`, `longrun_aquaplanet_amip` and `diagnostic_edmfx_aquaplanet`
- Removes `Float32` in some longrun configs as that is the default.

## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

File: config/perf_configs/jet_n_failures.yml [hunk 1]
```diff
@@ -1 +1,2 @@
 job_id: "jet_n_failures"
+implicit_diffusion: false
```
Comment (charleskawczynski) [inline config/perf_configs/jet_n_failures.yml:2]: @szy21, why was this added to this jet job?

Comment (szy21) [reply config/perf_configs/jet_n_failures.yml:2]: Oh, if it's not clear, `implicit_diffusion` has been always false in this jet job. But I set it to true in the default perf config in this PR, so this line is to set it to false again.
